README.md

```diff
@@ -365,9 +365,20 @@ hellosign.embedded.getSignUrl(signature_id)
 
 #### Get an embedded template's edit URL
 ````javascript
+GET
```

Comment:
Being that this is inside a `javascript` block, it would be prudent to use comments for these headings

```suggestion
// GET
```